Mathlib/Probability/Distributions/Gaussian.lean

```diff
@@ -347,6 +348,38 @@ lemma gaussianReal_mul_const {X : Ω → ℝ} (hX : Measure.map X ℙ = gaussian
 
 end Transformations
 
+open Measurable Real
+
+variable {Ω : Type} [MeasureSpace Ω] {μ : ℝ} {v : ℝ≥0} {X : Ω → ℝ}
+
+theorem mgf_gaussianReal (hv : v ≠ 0) (hX : Measure.map X ℙ = gaussianReal μ v) (t : ℝ) :
```

Comment:
The result is also true for `v = 0`, right? The proof will be different and probably needs one more lemma about the mgf of a Dirac.